Formula/g/gnupg@2.5.rb

```diff
@@ -0,0 +1,77 @@
+class GnupgAT25 < Formula
+  # "this 2.5 series is fully supported and thus ready for production use"
+  #   https://lists.gnupg.org/pipermail/gnupg-announce/2025q4/000499.html
```

Comment:
Given this announcement we should probably update the `gnupg` formula to 2.5 (and consider adding a `gnupg@2.4`) instead.

Reply:
A few days ago, we decided not to. https://github.com/Homebrew/homebrew-core/pull/257408

The status has not changed, the announcement as already that way for previous releases. But the homepage still lists it as “devel”:

<img width="827" height="190" alt="Capture d’écran 2025-12-10 à 10 03 24" src="https://github.com/user-attachments/assets/29720ceb-0afb-42d6-9f9f-fb984b31b8ce" />